MSAL/src/requests/MSALRequestParameters.h

```diff
@@ -69,4 +71,6 @@
 #pragma mark Methods
 - (void)setScopesFromArray:(NSArray<NSString *> *)array;
 - (void)setCloudAuthorityWithCloudHostName:(NSString *)cloudHostName;
+- (void)setClaims:(NSString *)claims NS_UNAVAILABLE;
```

Comment:
can we just remove the property claims instead?

Reply:
addressed by marking claims property to be readonly :)